src/CommandLineUtils/CommandArgument.cs

```diff
@@ -54,6 +56,22 @@ public CommandArgument()
         /// </summary>
         public string? Value => Values.FirstOrDefault();
 
+        /// <summary>
+        /// The default value of the argument.
+        /// </summary>
+        public string? DefaultValue
+        {
+            get => _defaultValue;
+            set
+            {
+                _defaultValue = value;
+                if (value != null)
+                {
+                    Values.Add(value);
+                }
```

Comment:
This is close, but seems not quite right. I think the behavior users would expect is:


command line | `arg.DefaultValue` | `arg.Values`
--|--|--
`cmd`  | `"arg1"`|  `{"arg1"}` 
`cmd arg2`  | `"arg1"` |  `{"arg2"}` 


But what is implemented here is:



command line | `arg.DefaultValue` | `arg.Values`
--|--|--
`cmd`  | `"arg1"` |  `{"arg1"}` 
`cmd arg2`  | `"arg1"` |  `{"arg1", "arg2"}`